src/components/tasks/TaskCommandLogs.tsx

```diff
@@ -13,13 +13,21 @@ import createStyles from '@mui/styles/createStyles';
 import withStyles from '@mui/styles/withStyles';
 import GetApp from '@mui/icons-material/GetApp';
 import Fab from '@mui/material/Fab';
-import { TaskCommandLogsTailQuery } from './__generated__/TaskCommandLogsTailQuery.graphql';
-import { TaskCommandStatus } from './__generated__/TaskCommandList_task.graphql';
+import {
+  TaskCommandLogsTailQuery,
+  TaskCommandLogsTailQueryResponse,
+} from './__generated__/TaskCommandLogsTailQuery.graphql';
+import { TaskCommandStatus, TaskCommandType } from './__generated__/TaskCommandList_task.graphql';
 
 function logURL(taskId: string, command) {
   return 'https://api.cirrus-ci.com/v1/task/' + taskId + '/logs/' + command.name + '.log';
 }
 
+function cacheURL(taskId: string, executionInfo) {
+  let cacheKey = executionInfo.cacheRetrievalAttempts.hits[0].key;
```

Comment:
I think is not accurate in case there are multiple `cache` instruction and/or [HTTP cache](https://cirrus-ci.org/guide/writing-tasks/#http-cache) is used.

Let's just find first `hit` with a `key` that stars with `command.name`. I think this is the most accurate because we know that a `key` is constructed like `<COMMAND NAME>-<FINGERPRINT>`.

Reply:
I see that `key` is a `string` starts with 'node_modules'